### PR TITLE
Libyt plummer example

### PR DESCRIPTION
**Pull request by cindytsai**
# libyt Plummer Example
A concrete example of how `gamer` uses `libyt`.

### CODE REVIEW

**Comment by hyschive** [comment]
@koarakawaii Could you help test it?

File: example/test_problem/Hydro/Plummer/inline_script.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import yt
+
+yt.enable_parallelism()
+
+def yt_inline():
+    # Get data
+    ds = yt.frontends.libyt.libytDataset()
+
+    # Do ProjectionPlot to field Cloud0.
+    sz = yt.ProjectionPlot(ds, 'z', ('gamer', 'Cloud0'), center='c')
```
Comment (koarakawaii) [inline example/test_problem/Hydro/Plummer/inline_script.py:10]: Do we need to fix the color bar?

File: example/test_problem/Hydro/Plummer/inline_script.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    # Do ParticlePlot
+    par = yt.ParticlePlot(ds, 'particle_position_x', 'particle_position_y', 'particle_mass', center='c')
```
Comment (koarakawaii) [inline example/test_problem/Hydro/Plummer/inline_script.py:13]: Do we need to fix the color bar?

File: example/test_problem/Hydro/Plummer/inline_script.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    if yt.is_root():
+        sz.save()
+        par.save()
+
+def yt_inline_inputArg( fields ):
+    # Get data
+    ds = yt.frontends.libyt.libytDataset()
+
+    # Do ProjectionPlot to fields input by gamer
+    sz = yt.ProjectionPlot(ds, 'z', fields, center='c')
```
Comment (koarakawaii) [inline example/test_problem/Hydro/Plummer/inline_script.py:24]: Do we need to fix the color bar?

**Review by koarakawaii — COMMENTED** [review]
The code works good and the tutorial is easy to understand :smiley: If the color bar is not an issue, I think we can merge it into `GAMER`~

**Comment by hyschive** [comment]
@cindytsai Thanks for the contribution!
@koarakawaii Thanks for helping review it! I don't think we need to fix the color bar. I'll merge it right away.

**Review by hyschive — APPROVED** [review]

**Pull request merged**